src/ChatGPTBrowserClient.js

```diff
@@ -209,4 +235,51 @@ export default class ChatGPTBrowserClient {
             details: result,
         };
     }
+
+    genTitle(event) {
+        const debug = this.options.debug;
+        if (debug) {
+            console.log('Generate title: ', event);
+        }
+        if (!event || !event.conversation_id || !event.message || !event.message.id) {
+            return;
+        }
+
+        try {
+            const conversationId = event.conversation_id;
+            const messageId = event.message.id;
+            
+            const url = 'https://chat.openai.com/backend-api/conversation/gen_title/' + conversationId;
+            const opts = {
+                method: 'POST',
+                headers: {
+                    'Content-Type': 'application/json',
+                    Authorization: `Bearer ${this.accessToken}`,
+                    Cookie: this.cookies || undefined,
+                },
+                body: JSON.stringify({
+                    message_id: messageId,
+                    model: this.model,
+                }),
+            };
+
+            if (debug) {
+                console.log('gen title fetch opts: ', opts);
+            }
+            
+            if (this.options.proxy) {
+                opts.dispatcher = new ProxyAgent(this.options.proxy);
+            }
+
+            fetch(url, opts).then((ret) => {
+                if (debug) {
+                    ret.json().then((data) => {
+                        console.log('Gen title response: ', data);
+                    });
+                }
+            }).catch(console.error);
+        } catch(err) {
+            console.error(err);
+        }
```

Comment:
This should also be unnecessary due to the `.catch` above.

Reply:
This block is for debug that was not removed

Reply:
@hyperzlib let me know when you fix this one and I'll test it

Reply:
Updated.